src/query-compiler/compiled-query.ts

```diff
@@ -9,11 +9,11 @@ export interface CompiledQuery<O = unknown> {
 }
 
 export const CompiledQuery = freeze({
-  raw(sql: string): CompiledQuery {
+  raw(sql: string, ...parameters: unknown[]): CompiledQuery {
```

Comment:
Anyone using this directly with parameters will probably have them in an array. Let's not force consumer to spread.

```suggestion
  raw(sql: string, parameters: unknown[] = []): CompiledQuery {
```